Tools/Incinerator/GLTFLoader.cpp

```diff
@@ -171,6 +194,68 @@ bool ParseAndIncinerate(const uint8_t* gltfData,
 		std::ofstream outputFile(destinationFolder + meshName, std::ios::binary | std::ios::out | std::ios::trunc);
 		outputFile.write(reinterpret_cast<const char*>(memstream.GetData()), memstream.GetDataSize());
 	}
+
+	for (const auto& materialIndex : materialToIncinerate)
+	{
+		Zmey::Graphics::MaterialDataHeader dataHeader;
+		// Default data
+		dataHeader.BaseColorFactor = Zmey::Color(1.0f, 1.0f, 1.0f, 1.0f);

```

Comment:
sounds like a default ctor ?

Reply:
Yes, but I didn;t want to put the logic in the ctor. It is possible that later the default material could include textures and stuff like that. So i prefered to leave in the implementation.